unreal_qt_stylesheet/__init__.py

```diff
@@ -0,0 +1 @@
+from unreal_qt_stylesheet.main import *
```

Comment:
can we change the package name to be "unrealStylesheet"?

Reply:
gotcha, I was thinking it to be consistent with the repo name...
I agree going with snake case, how's 'unreal_stylesheet'? don't want the package name to be too long.

Reply:
will change to that

Reply:
And if you want consistancy with the repo name,
you can always rename the repo in GitHub. e.g. to the pypi recommended name of `unreal-stylesheet`.
Anyone who still visits or clones `UnrealStylesheet` will be [redirected](https://github.blog/2013-05-16-repository-redirects-are-here/) by GitHub to the new URL